__std__/nix/toolchain/packages/check-the-flake.nix

```diff
@@ -0,0 +1,29 @@
+{ inputs, cell }:
+
+inputs.nixpkgs.writeShellApplication {
+  name = "check-the-flake";
+  runtimeInputs = [ inputs.nixpkgs.nix ];
+  text = ''
+    nix develop .#haskell-shell --build
```

Comment:
This seems a bit manual...

Reply:
Yes. I'm automating this in the next PR.

Reply:
Persoectively there may be thaought of parallelizing these things to get the CI total time metric down.

SRE R&D is working on this... (I think :smile:).

Just a headsup, not relevant right now.

Reply:
Well, we need a CI solution to be sure. I'm not sure we have anything that will replace our `ci.nix` so far?

Reply:
Nothing, yet, unfortunately. I just wanted to anchor a headsup, here...